packages/contentstack-bulk-publish/src/commands/cm/bulk-publish/unpublished-entries.js

```diff
@@ -1,100 +1,15 @@
 const { Command, flags } = require('@contentstack/cli-command');
```

Comment:
You can remove this file and add deprecation in package.json and use alias for old command in new command

Reply:
These changes have been made!